api/openapi.yaml

```diff
@@ -353,6 +353,42 @@ paths:
           description: Endpoint not found
         '405':
           description: 'Invalid method, only GET,POST is allowed'
+
+  /bmc/creds:
+    parameters:
+      - in: query
+        name: targets
+        required: false
+        description: Comma separated list of XNames.  No query string results in fetching creds for all known BMCs.
+        schema:
+          $ref: '#/components/schemas/xname_list'
+      - in: query
+        name: type
```

Comment:
if someone specifies an xname and a type is it a logical AND ?  eg if they say x100c0s0r0b0 NodeBMC will it come back NULL? or 404?

Reply:
Yes, logical AND.